packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-design.md

```diff
@@ -0,0 +1,48 @@
+---
+id: Design for accessibility
+section: accessibility
+---
+
+As described in our [accessibility fundamentals](/accessibility/accessibility-fundamentals), users may interact with your product through a variety of assistive technologies. In addition to [developing for accessibility](/accessibility/accessibility-development), you must also incorporate it into your product's UI design. 
+
+
+
+## Color and contrast
+
+Color and contrast go hand-in-hand for visual design accessibility. 
+
+For example, you should consider contrast when using colors commonly affected by color perception deficiency (such as those described in [color perception by W3C](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Overview_of_Low_Vision#Color_Perception)). Different colors in a UI should be distinguishable for users across the color perception deficiency spectrum, typically by ensuring that each color has a different contrast weight. 
+
+Designing with contrast in mind also benefits users with light sensitivity.
+
+[PatternFly's colors](/design-foundations/colors) pass color contrast tests, so long as you follow our recommendations. 
+
+## Content 
+
+Your text content should also consider the needs of users with low vision. For example, links should be descriptive and concise, so that users can quickly identify their purpose. 
+
+To help simplify and strengthen the way that users process content, it is also important to carefully consider your text size, font, letter spacing, margins, and spacing. PatternFly supports these considerations through our [typography](/design-foundations/typography) and [spacing](/design-foundations/spacers) guidelines.
+
+## Understanding low vision 
+
+In addition to following the previous recommendations, it can be helpful to develop an understanding of low-vision users, so that their needs are top of mind as you design.
+
+**Low vision** is frequently defined as a visual impairment that impedes a person’s ability to perform everyday tasks, which cannot be corrected with glasses, contacts, medicine, or surgery. 
+
+There are 4 types of limitations as a result of low vision.
+
+### 1. Visual acuity 
```

Comment:
Nit but not a blocker, similar to another comment from one of the other PRs but wondering if we need the numbers for these 4 sections

Reply:
yep - good call!